### ISSUE TITLE
Error message when changing Intensity

### ISSUE THREAD

**Issue by vodogg25** [natekspencer/hacs-pura#76]
### The problem

The scents are working fine and I can turn them all on through HA with no issues. For some reason, when i try to change the intensity, I am getting an error message that states " Failed to perform the action select/select_option. 400 Client Error: Bad Request for url: https://trypura.io/mobile/api/devices/D48AFC8E9844/intensity" After several attempts changing slots back and forth and inensity, it will eventually save the correct intensity I'm trying to select but i get this message multiple times first. 

### What version of Home Assistant Core are you running?

	core-2024.11.3

### What was the last working version of Home Assistant Core?

	core-2024.11.3

### Diagnostics information

[config_entry-pura-01JBDEESG55A2XFYEGWER38ZTH.json](https://github.com/user-attachments/files/17880173/config_entry-pura-01JBDEESG55A2XFYEGWER38ZTH.json)

**Comment by natekspencer** [comment]
Does this happen on every diffuser or one in particular? And if so, what is the device model giving you issues?

**Comment by vodogg25** [comment]
Hey buddy - I am using the pura 4. I have several of them and they all are doing it.

**Comment by evrowe** [comment]
I also have the Pura 4 and have encountered the same issue. However, I typically only have to set the intensity once, and even after the error shows, the diffuser will switch on using the desired slot and intensity a few seconds later. Without actually looking at the code yet, I would guess/assume this is a caching issue or a delay in communications back and forth with Pura's API.

For me it's more of an experience issue than it is an actual usability issue 🤷

**Comment by natekspencer** [comment]
Can you enable debugging and then adjust the intensity to capture the error and other info being sent so I can better track this down? Also, are you adjusting intensity while on a timer, schedule or always on modes?

**Comment by evrowe** [comment]
Debug log attached.

I am only adjusting the intensity for "always on" mode, or rather, to enable the diffuser at all; I don't have schedules set up, and I can't seem to get timer controls to work for the Pura 4 in HA (my `sensor.office_diffuser_controller` only shows a state of "default"). I could try starting a timer from the Pura app and seeing what happens when adjusting intensity within HA.

[home-assistant_pura_2025-01-09T18-15-21.785Z.log](https://github.com/user-attachments/files/18365571/home-assistant_pura_2025-01-09T18-15-21.785Z.log)

**Comment by natekspencer** [comment]
> Debug log attached.

@evrowe, yeah, yours is unfortunately different from the original error and, like you mentioned, a result of the Pura not updating the status right away after turning it on. Thus it still thinks it isn't running.

I need to document the `pura.start_timer` service (and also update to use the new subtle, medium, strong settings), but you can try something like this right now:
```yaml
action: pura.start_timer
data:
  slot: "2"
  duration:
    hours: 0
    minutes: 10
    seconds: 0
  intensity: 6
target:
  entity_id: select.living_room_diffuser_fragrance
```

**Comment by natekspencer** [comment]
@vodogg25 release 1.2.2 should resolve your issue
